Makefile

```diff
@@ -180,6 +184,9 @@ pwqfilter.o: pwqfilter.c passwdqc_filter.h passwdqc.h
 .c.o:
 	$(CC) $(CPPFLAGS_lib) $(CFLAGS_lib) -c $*.c
 
+$(PKGCONFIG):
+	sed -e "s|@VERSION@|$(VERSION)|g" $@.in > $@
```

Comment:
Also, $(PKGCONFIG) should depend on its template.

Reply:
done